### ISSUE TITLE
autoapi subtitle looks wrong

### ISSUE THREAD

**Issue by humitos** [rtfd/sphinx-notfound-page#48]
I'm not sure what I'm doing wrong with `sphinx-autoapi` but the subtitle looks weird and needs to be fixed.

![Screenshot_2019-06-07_14-49-03](https://user-images.githubusercontent.com/244656/59105116-8210fe80-8933-11e9-8f77-422f4f53216f.png)

I suppose the "Create a 404.html page" should be at the same level that the paragraph and maybe in bold. Also, "404.html" should be rendered as code.

The page affected by this is https://sphinx-notfound-page.readthedocs.io/en/latest/autoapi/notfound/extension/index.html

**Comment by acozine** [comment]
I looked at the [code that generates this docstring](https://github.com/rtfd/sphinx-notfound-page/blob/7276e3809209e1d6819f83065558bc5f3ba718a5/notfound/extension.py#L19). I don't see anything obvious wrong with it, but I see that line 19 is being exported in the same style as the heading. It's weird. I tried adding an extra blank line between the `"""` and line 19, but that didn't fix this issue. 

If you know of other projects that are using `autoapi`, post links. I can look at some examples next week and maybe the fix will be obvious.
